### ISSUE TITLE
Heading Levels unpredictable

### ISSUE THREAD

**Issue by adplusverb** [thesmythgroup/vue-toolkit#25]
## Describe of the Issue

Heading levels are not predictable and affect accessibility. For example, in the doc site the section title is an h2 (and should be an h1). The accordion component is hard-coded to be an h3, which would skip. I don't necessarily have a solution to suggest, but just pointing out that based on where an accordion or other component is used it might break the Heading tree.

<img width="716" alt="image" src="https://user-images.githubusercontent.com/6987835/136860651-e8eadc12-4394-4f6f-afeb-e75a1532c0c1.png">
